File: WindowsDevScripts/tests/go/hello.go
// Hello-world probe for the Go flow.
//
// `go run` accepts a single .go file directly without a surrounding module,
// so this file is intentionally standalone (no go.mod). If the toolchain
// install was incomplete, `go run` would fail and the harness would flag
// the flow broken.

package main

import "fmt"

func main() {
	fmt.Println("Hello, world!")
}

File: WindowsDevScripts/tests/java/Hello.java
// Hello-world probe for the Java flow.
//
// Exercised via JDK 11+'s single-file source-code launcher (JEP 330):
// `java tests/java/Hello.java` compiles and runs this file in one step,
// so no explicit `javac` build is needed. If the JDK install was
// incomplete, the launcher would fail and the harness would flag the
// flow broken.

public class Hello {
    public static void main(String[] args) {
        System.out.println("Hello, world!");
    }
}
